src/components/product-card/types.ts

```diff
@@ -0,0 +1,15 @@
+export interface ProductCardProps {
+  text: string;
+  city: string;
+  margin?: number;
+  avatar?: string;
+  picture: string;
+  buttonText: string;
+  onClick: () => void;
+  isFavorite?: boolean;
+  inboxMessage?: number;
+}
+
+export interface InboxMessageProps {
+  inboxMessage: number;
+}
```

Comment:
This interface you use in src/components/inbox-message/index.tsx
for this it was created in
src/components/inbox-message/types.ts
maybe it's not needed here?

Reply:
That is how it should be